submarine-test/test-k8s/src/test/java/org/apache/submarine/rest/JobManagerRestApiIT.java

```diff
@@ -60,7 +60,7 @@ public void testCreateJob() throws Exception {
 
     Gson gson = new Gson();
     JsonResponse jsonResponse = gson.fromJson(responseBodyAsString, JsonResponse.class);
-    assertEquals("Response code should be 202 ",
-        Response.Status.ACCEPTED.getStatusCode(), jsonResponse.getCode());
+    assertEquals("Response code should be 406 ",
```

Comment:
I'm kind of confusing with this one and the JobManagerRestApiTest (AbstractSubmarineServerTest). Both the two seem to start submarine-server locally and do REST queries.
If we'd prefer this JobManagerRestApiIT as integration and end-to-end test, then we should figure out a way to mock the submarine server instead of start a real submarine server in the JobManagerRestApiTest. Thoughts? @jiwq , @liuxunorg , @yuanzac